src/registration/migrations/0003_registration_profile_status.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('registration', '0002_default_supplement'),
```

Comment:
**NOTE**
It depends on `0002_default_supplement` but actual migration won't performed so it's ok. https://github.com/lambdalisue/django-inspectional-registration/blob/master/src/registration/migrations/0002_default_supplement.py#L14